### PR TITLE
Update release profile for consistency

### PR DESCRIPTION
**Pull request by philipsens**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -162,6 +185,16 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
```
Comment (evandongen) [inline pom.xml:165]: Can this be updated as well? 1.6 is from 2020, 3.2.4 is current. But perhaps this is out of scope for this PR.

File: pom.xml [hunk 2]
```diff
@@ -172,31 +205,36 @@
...
-            </configuration>
-          </plugin>
           <plugin>
             <groupId>org.sonatype.plugins</groupId>
             <artifactId>nexus-staging-maven-plugin</artifactId>
-            <version>1.6.8</version>
+            <version>1.6.13</version>
             <extensions>true</extensions>
             <configuration>
               <serverId>ossrh</serverId>
```
Comment (evandongen) [inline pom.xml:200]: Is it possible to use a maven property for this value instead of duplicating it multiple times? Same for 'frankframework'.

File: pom.xml (hunk 1 above)
Comment (philipsens) [reply pom.xml:165]: That could be useful. We had to debug it for a while, and then we left it at this version. It should be tested first before being updated. Maybe that would be better in a follow-up pull request.

File: pom.xml (hunk 2 above)
Comment (philipsens) [reply pom.xml:200]: There is no other location where this plugin is used, are you referring to something else?

Comment (evandongen) [reply pom.xml:200]: I was referrring to the value 'ossrh' (serverId below), it's used multiple times.

Comment (philipsens) [reply pom.xml:200]: I'll remove the distribution management

File: pom.xml (hunk 1 above)
Comment (philipsens) [reply pom.xml:165]: I'll try the new version first. So I'll draft this PR until that's done.

**Review by nielsm5 — APPROVED** [review]
